### PR TITLE
Some account transaction getter changes

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

**Review by Alex6323 — DISMISSED** [review]

File: bindings/core/src/response.rs [hunk 1]
```diff
@@ -331,11 +331,11 @@ pub enum Response {
...
     Balance(AccountBalanceDto),
     /// Response for
-    /// - [`GetIncomingTransactionData`](crate::method::AccountMethod::GetIncomingTransactionData),
-    IncomingTransactionData(Option<Box<(TransactionId, TransactionDto)>>),
+    /// - [`GetIncomingTransaction`](crate::method::AccountMethod::GetIncomingTransaction),
+    IncomingTransaction(Option<Box<TransactionDto>>),
     /// Response for
     /// - [`IncomingTransactions`](crate::method::AccountMethod::IncomingTransactions),
-    IncomingTransactionsData(Vec<(TransactionId, TransactionDto)>),
+    IncomingTransactions(TransactionDto),
```
Comment (qrayven) [inline bindings/core/src/response.rs]: Shouldn't be a vector of transactions?

Comment (Thoralf-M) [inline bindings/core/src/response.rs]: Remove the variants and just return the Transaction/Transactions

Comment (thibault-martinez) [reply bindings/core/src/response.rs]: Done

Comment (thibault-martinez) [reply bindings/core/src/response.rs]: Fixed by addressing Thoralf's comment

File: .changes/getIncomingTransaction.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"wallet-nodejs-binding": patch
+---
+
+Rename `getIncomingTransactionData` to `getIncomingTransaction`.
```
Comment (Thoralf-M) [inline .changes/getIncomingTransaction.md:5]: Also changed the return type of getIncomingTransactions?

Comment (Thoralf-M) [reply .changes/getIncomingTransaction.md:5]: Ah no, that's doesn't exist there

**Review by Thoralf-M — DISMISSED** [review]

**Review by Alex6323 — DISMISSED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**